### PR TITLE
v7.2.0

### PR DESCRIPTION
**Pull request by connorjclark**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,56 @@
...
+
+- @1d0n7kn0w
+- Benajmin Franz @BennyAlex
+
+## Core
+
+* emulation: bump chrome versions ([#12104](https://github.com/GoogleChrome/lighthouse/pull/12104))
+* unsized-images: ignore non-network SVGs ([#12120](https://github.com/GoogleChrome/lighthouse/pull/12120))
+* preload: do not warn for cache loads ([#12122](https://github.com/GoogleChrome/lighthouse/pull/12122))
+* url: redirects-http => n/a for localhost. refactor url special cases ([#11766](https://github.com/GoogleChrome/lighthouse/pull/11766))
```
Comment (brendankenny) [inline changelog.md:19]: don't even know what to do with this commit message :)

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+* fr: add navigation phase + trace gatherer ([#12098](https://github.com/GoogleChrome/lighthouse/pull/12098))
+* fr: add DevtoolsLog artifact ([#12074](https://github.com/GoogleChrome/lighthouse/pull/12074))
+
+## Deps
+
+* update snyk snapshot ([#12132](https://github.com/GoogleChrome/lighthouse/pull/12132))
+
+## Clients
+
+* lr: should skip uses-http2 audit for mobile ([#12115](https://github.com/GoogleChrome/lighthouse/pull/12115))
```
Comment (brendankenny) [inline changelog.md:34]:
```suggestion
* lr: skip uses-http2 audit for mobile ([#12115](https://github.com/GoogleChrome/lighthouse/pull/12115))
```
maybe?

File: changelog.md [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+
+## Tests
+
+* sync devtools tests ([#12127](https://github.com/GoogleChrome/lighthouse/pull/12127))
+
+## Misc
+
+* treemap: first pass ([#11832](https://github.com/GoogleChrome/lighthouse/pull/11832))
+* add LIGHTHOUSE_FLAGS to gcp script. tweak extract script ([#12100](https://github.com/GoogleChrome/lighthouse/pull/12100))
+* rename lantern master values to baseline values ([#10950](https://github.com/GoogleChrome/lighthouse/pull/10950))
```
Comment (brendankenny) [inline changelog.md:49]:
```suggestion
* rename lantern baseline values ([#10950](https://github.com/GoogleChrome/lighthouse/pull/10950))
```

**Review by brendankenny — APPROVED** [review]
LGTM!

File: changelog.md (hunk 1 above)
Comment (connorjclark) [reply changelog.md]: i just changed it

**Pull request merged**